pkg/util/kubernetes/hostinfo/tags_test.go

```diff
@@ -8,11 +8,85 @@
 package hostinfo
 
 import (
+	"reflect"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+
+	configutils "github.com/DataDog/datadog-agent/pkg/config/utils"
+	"github.com/DataDog/datadog-agent/pkg/util/kubernetes"
 )
 
+type mockMetadataAsTags struct {
+	nodeLabelsAsTags      map[string]string
```

Comment:
```suggestion
    configutils.MetadataAsTags
	nodeLabelsAsTags      map[string]string
```


To avoid including unimplemented methods like GetPodLabelsAsTags in your mock implementation, you can embed the interface that defines those methods. This way, you only need to implement the methods you actually use in your tests.

Reply:
Oh great!
I didn't know this is possible. 

Thanks 😄